plugins/available/sudo.plugin.bash

```diff
@@ -0,0 +1,15 @@
+cite about-plugin
+about-plugin 'Toggle sudo at the beginning of the current or the previous command'
+
+function sudo-command-line() {
```

Comment:
Since the `READLINE` functionality requires at least Bash v4, please add a check for a Bash version that is greater or equal than 4. See here for an example: https://github.com/Bash-it/bash-it/blob/f2c07f5143b61c3fdb549bfe3012a684d558ba6d/completion/available/system.completion.bash#L24